src/pages/LoginPage.js

```diff
@@ -56,6 +56,11 @@ class LoginPage extends React.Component {
         });
     };
 
+    handleRegister() {
```

Comment:
Change this to 'handleRegister = () => {'
instead of 'handleRegister() {'
This won't access props otherwise(I believe)